inc/Widget/SmailyWidget.php

```diff
@@ -107,10 +107,67 @@ public function widget( $args, $instance ) {
+						<input class="smaily-layout-1-input" type="text" name="email" value="" placeholder="' . esc_attr( $instance['email_field_placeholder'] ) . '" />
+						<button class="ui pink basic button smaily-button" type="submit" style="background-color: ' . esc_attr( $instance['submit_button_color'] ) . ' ; color: ' . esc_attr( $instance['submit_button_text_color'] ) . '" >' . esc_attr( $instance['submit_button_text'] ) . '</button>
+					</div>';
+					break;
+				// Layout 2.
+				case 'layout-2':
+					$layout = '<div class="smaily-layout-2-container">
+						<input type="text" name="email" value="" placeholder="' . esc_attr( $instance['email_field_placeholder'] ) . '" />
+						<button class="ui pink basic button smaily-button" type="submit" style="background-color: ' . esc_attr( $instance['submit_button_color'] ) . ' ; color: ' . esc_attr( $instance['submit_button_text_color'] ) . '" >' . esc_attr( $instance['submit_button_text'] ) . '</button>
+					</div>';
+					break;
+				// Layout 3.
+				case 'layout-3':
+					$layout = '<div class="smaily-layout-3-container">
+						<input class="smaily-layout-3-input" type="text" name="email" value="" placeholder="' . esc_attr( $instance['email_field_placeholder'] ) . '" />
+						<p>
+							<button class=" ui pink basic button smaily-button" type="submit" style="background-color: ' . esc_attr( $instance['submit_button_color'] ) . ' ; color: ' . esc_attr( $instance['submit_button_text_color'] ) . '" >' . esc_attr( $instance['submit_button_text'] ) . '</button>
+						</p>
+					</div>';
+					break;
+				// Layout 4.
+				case 'layout-4':
+					$layout = '<div class="smaily-layout-4-container">
+						<input class="smaily-layout-4-input-email" type="text" name="email" value="" placeholder="' . esc_attr( $instance['email_field_placeholder'] ) . '" />
+						<input class="smaily-layout-4-input-name" type="text" name="name" value="" placeholder="' . esc_attr( $instance['name_field_placeholder'] ) . '" />
+						<button class="ui pink basic button smaily-button" type="submit" style="background-color: ' . esc_attr( $instance['submit_button_color'] ) . ' ; color: ' . esc_attr( $instance['submit_button_text_color'] ) . '" >' . esc_attr( $instance['submit_button_text'] ) . '</button>
+					</div>';
+					break;
+				// Layout 5.
+				case 'layout-5':
+					$layout = '<div class="smaily-layout-5-container">
+						<input class="smaily-layout-5-input" type="text" name="email" value="" placeholder="' . esc_attr( $instance['email_field_placeholder'] ) . '" />
+					</div>
+					<p>
+						<input type="text" name="name" value="" placeholder="' . esc_attr( $instance['name_field_placeholder'] ) . '" />
+					</p>
+					<p>
+						<button class="ui pink basic button smaily-button" style="background-color: ' . esc_attr( $instance['submit_button_color'] ) . ' ; color: ' . esc_attr( $instance['submit_button_text_color'] ) . '" >' . esc_attr( $instance['submit_button_text'] ) . '</button>
+					</p>';
```

Comment:
Layout 5 does not follow the structure of other layouts, and it breaks the input field positioning (see attached image).
![Screenshot from 2021-06-08 16-12-46](https://user-images.githubusercontent.com/6755852/121191603-cada5580-c874-11eb-9234-67f19d4a3495.png)